cmd/deploy/deploy.go

```diff
@@ -434,6 +393,7 @@ func getProxyHandler(name, token string, clusterConfig *rest.Config) (http.Handl
 				rw.WriteHeader(500)
 				return
 			}
+			defer r.Body.Close()
```

Comment:
duplicated defer?

Reply:
yep, it was introduced by another PR and merge didn't detect any conflict